install.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 set -uo pipefail
-mpremote mkdir :/flash/sys/apps/salzamt-badge
+sudo mpremote mkdir :/sd/apps/salz
```

Comment:
Ich würd in die Readme "Call `sudo ./install.sh` reinschreiben und die `sudo`s rausschmeißen